### ISSUE TITLE
NLL in Temperature scaling

### ISSUE THREAD

**Issue by dsvrsec** [markus93/NN_calibration#10]
In temperature scaling code,negative log likelihood is considered not considered as loss function where as only Log likelihood is considered.Is there any reason or is it an error

**Comment by markus93** [comment]
Hey!

Actually, it is the same thing, the log loss is implemented as negative log-likelihood of a logistic model. More information in documentation of sklearn in [here](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.log_loss.html).
